### DISCUSSION TITLE
Can use a junction table in Doctrine as an entity

### DISCUSSION THREAD

**Discussion by gourdina** [symfony/symfony#46917] [Q&A]
Hi,
I have 4 entities: User, City, Service, Event
City has a relationship (ManyToMany) with Service. => The junction table:  city_service(city_id, service_id)
User also has a relationship (ManyToMany) with City and Service.
User has a relationship (OneToMany) with Event.

But there are problems, for example:
John works in Service(sales) in City(Paris and Lyon), but also works in Service(after-sales) in City(Paris), How can know whether he works in Service(after-sales) in City(Lyon) ?

My question is: 
Can use the junction table (city_service) for building another junction table with User (ManyToMany)?

Any advice will be welcome.

**Comment by MatTheCat** [comment]
The ` city_service` table should be one of your entities. I don’t know about your domain but it could be `Store` e.g. as one would provide services in a given location. You can then link `User` and `Store`, which makes sense.
